server/index.js

```diff
@@ -1,17 +1,22 @@
 'use strict';
 
-const path = require('path')
+const path = require('path');
 
 const express = require('express');
 const app = require('express')();
 const http = require('http').Server(app);
 const io = require('socket.io')(http);
 
-const board = require('../client/assets/js/board.js')
+import Board from '../client/assets/js/board.js';
 
 const totalDuration = 5000;
 const gameDuration = 3000;
 
+var applesOnBoard = [];
+var b = new Board();
```

Comment:
This part is awkward. 
First off, the board already has an apple array https://github.com/ecvdbdx1617/MultiSnake/blob/8fd6f5b26cb67c0ef520bf1796f15d15c7c4cf7d/client/assets/js/board.js#L16

then passing an empty array so it's filled by a function you call is a strange practice.
Maybe having `generateApples` as a method of the board would be preferable